### ISSUE TITLE
glide with obgaektify not working

### ISSUE THREAD

**Issue by m-kumaran** [kdabir/glide#12]
Hi

Thanks for the excellent framework to ease the GAE development.

I found one problem, when I tried to include obgaektify (obgaektify.appspot.com)  in a glided app, it is not recognising the model classes for registering.
It throws exception with a message "No such property". But when I try with plain groovy app it works.

Will it be possible to include  a sample app with Objectified models?
Basically I am trying to persist  Parent/List<Child>  and the sample didn't guide on this.

Thanks in advance.

Cheers
Kumaran
